### DISCUSSION TITLE
Long term support

### DISCUSSION THREAD

**Discussion by tyleralbee** [RobinTail/express-zod-api#1568] [General]
I really love this library. I think the abstractions are super well thought-out, it combines a few of my favorite tools, and I expect it'll fit well into my project.

I looked at the contribution graph and I couldn't help but notice that the owner of the repo has created the vast majority of it themselves. That's awesome, and I really respect that. I was hoping to gauge whether or not they expect to continue doing so, if they are looking for other maintainers, and whether or not they feel it's ready for a production application.

If anyone wants to chime in and let me know whether or not they're using this in production that'd be great too.

**Comment by RobinTail** [comment]
Thank you for appreciation, @tyleralbee .

> whether or not they expect to continue doing so

You can expect the order to be maintained.

> whether or not they feel it's ready for a production application

I may have different feelings, but none of them are guarantees. This is also expressed [in the license text](https://github.com/RobinTail/express-zod-api/blob/63faf35578464a2fe2f4cc4cfb63fcd4333d6160/LICENSE#L15). I wanna share with you my personal opinion on this topic: you don't have to rely on someone else's promises because that doesn't create a real confidence. Instead, you can find that confidence within self-reliance. To do this, you can study the open source code and the history of fixing its bugs, make own conclusions, try it in action, write tests and develop a plan in case of unforeseen consequences. In particular, you can learn how to fork repositories in order to create modifications of the original solutions, independent of their original authors and external circumstances. Once you realize that you are capable of solving a problem with third-party software on your own, you will actually no longer need external affirmations for implementing your ideas and intentions. In this way, you can come to confident decision-making that comes with your own rather than scattered responsibility.
